src/status_im/contexts/wallet/send/input_amount/controlled_input_logic.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im.contexts.wallet.send.input-amount.controlled-input-logic
```

Comment:
This file duplicates a logic from `controlled-input/utils.cljs` because I didn't want to include that into refactoring.
Here all the functions operate only on `input-value`, not on `state` structure as in `controlled-input/utils.cljs`